redis/src/cluster_async/mod.rs

```diff
@@ -1372,13 +1401,22 @@ where
         res
     }
 
+    /// Update relevant slots in the slots map based on the moved_redirect address
+    pub(crate) async fn update_slots_for_redirect_change(
+        _inner: Arc<InnerCore<C>>,
+        _moved_redirect: Option<RedirectNode>,
+    ) {
```

Comment:
By default, I would make this function returning `Result<(), RedisError>` (assuming we have such error type `RedisError`)